.github/workflows/e2e-reusable.yml

```diff
@@ -141,8 +142,8 @@ jobs:
           # We have to provide custom ci-build-id key to make sure that this workflow could be run multiple times
           # in the same parent workflow
           ci-build-id: ${{ needs.prepare.outputs.uuid }}
-          spec: "/__w/n8n/n8n/cypress/${{ inputs.spec }}"
-          config-file: /__w/n8n/n8n/cypress.config.js
+          spec: '/__w/n8n/n8n/cypress/${{ inputs.spec }}'
+          config-file: /__w/n8n/n8n/cypress/cypress.config.js
```

Comment:
The e2e specs are now failing, we might have to add /cypress to the caching path similiar to /packages:
```
- name: Cache pnpm modules
        uses: actions/cache@v3
        with:
          path: |
            /github/home/.cache
            /github/home/.pnpm-store
            ./node_modules
            ./packages
          key: ${{ inputs.cache-key }}
          
```